src/Uno.UWP/Storage/Streams/DataWriter.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using System.IO;
+
+namespace Windows.Storage.Streams
+{
+
+	// currently only ctor without parameters is implemented,
+	// but some methods are already in form for both ctors in API
+
+
+	public partial class DataWriter : IDataWriter, IDisposable
+	{
+		// only one will be not null - depends on constructor used
+		private readonly MemoryStream _memStream = null;
+		private IOutputStream _outputStream = null;		// can be changed in DetachStream
+
+		public UnicodeEncoding UnicodeEncoding { get; set; }
+
+		public ByteOrder ByteOrder { get; set; }
+
+		public uint UnstoredBufferLength
+		{
+			get => (uint)(_memStream.Length - _memStream.Position);
+		}
+
+		// when someone will implement StoreAsync() method:
+		//public DataWriterStoreOperation StoreAsync()
+		//{

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5527039)